tests/data/abs_files/ftp/arxiv/papers/8080/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+
+[requires]
+python_version = "3.6"
```

Comment:
Why is there a Pipfile here?

Reply:
Ah, because I typed some pipenv in the wrong directory.

I expected it to work like npm or yarn and look up the tree find a project file.